src/catalog.ts

```diff
@@ -22,23 +22,26 @@ export class Catalog {
   }
 
   public static async fromStore(store: RDF.Store[]): Promise<Catalog> {
+    const properties =
+      '?dataset ?name ?creator ?creatorName ?creatorAlternateName ?distribution ?endpointUrl ?searchQuery ?lookupQuery ?alternateName';
     const query = `
       PREFIX schema: <http://schema.org/>
-        SELECT * WHERE {
+        SELECT ${properties} (GROUP_CONCAT(?url) as ?url) WHERE {
```

Comment:
So much less elegant query now, but this was needed to concatenate the prefix URLs into one binding.

Reply:
Ouch! Well, sometimes some ugliness is necessary to achieve greater beauty 😅

Perhaps add a comment to the query, to make clear why this construction is in place? It may not be that obvious to future readers or developers of the code?